tests/test-run.sh

```diff
@@ -74,6 +74,13 @@ assert_file_has_content hello_out '^Hello world, from a sandbox$'
 
 ok "hello"
 
+mkdir xrd
```

Comment:
The only difference between this and #4373 is that in this version of the test, we create the `XDG_RUNTIME_DIR`: otherwise the test fails with:

```
++ flatpak run --command=bash org.test.Platform -c 'echo $XDG_RUNTIME_DIR'
bwrap: Can't find source path : No such file or directory
```

I'm not sure why, but creating the `XDG_RUNTIME_DIR` is realistic and harmless.